salesagility/SuiteCRM
# m2m: fix SecurityGroup instance checking

Fix the way `M2MRelationship::add` method checks if given relation is added between `User` / `ACLRole` and `SecurityGroup` beans

## Description
When `User`, `ACLRole` or `SecurityGroup` is overloaded and the custom class is registered in `include/modules_override.php`  then user in no longer able to assign users/roles to security groups

## How To Test This
1. Overload any of the bean classes: `User`, `ACLRole`, `SecurityGroup`
2. Go to `Administration` -> `Security Groups` -> go to preview of any Security Group -> try to add User and/or Role to the security group.
3. Current behavior: 500 error when adding user or group. Expected behavior: user / role is added to security group

## Types of changes
- [x] Bug fix (Cannot add user/role to security group when any of the classes are overloaded)
